client/src/components/LiveTickerWrapper/index.js

```diff
@@ -1,14 +1,19 @@
 import { connect } from 'react-redux';
 
 import LiveTickerWrapper from './LiveTickerWrapper';
+import BigNumber from "bignumber.js";
 
+// TODO: read this from explorer-server, when it's ready
+const burnedTokens = new BigNumber(268785651.397875);
```

Comment:
Sadly, hardcoded for now... 

In order to read this value (the balance of `panacea10000...`), we need to change the call from frontend to backend. But, it causes many changes because this legacy codebase isn't modulated.

Reply:
:cry: